### PR TITLE
More space for keyword search

### PR DESCRIPTION
**Pull request by Gnito**
We wanted to show at least 20 characters on default mode and search was squeezed narrower on responsive layout between 1200px - 1270px (63% wasn't enough).
1200px - 1270px: no slide effect
1270++: slide effect

### CODE REVIEW

File: client/app/components/composites/SearchBar/SearchBar.css [hunk 1]
```diff
@@ -184,14 +184,18 @@
   }
 
   &:focus {
-    @media (--large-viewport) {
+    @media (--search-desktop-wide) {
```
Comment (mporkola) [inline client/app/components/composites/SearchBar/SearchBar.css]: What's this media query, and why the change?

Comment (Gnito) [inline client/app/components/composites/SearchBar/SearchBar.css]: We need to squeeze the search so that it fits 1200px layout. 
(So, that's 1270px breakpoint where we take 70 pixels away from Search component)
